tensorflow_lite_support/ios/task/vision/sources/TFLImageClassifier.m

```diff
@@ -79,8 +87,11 @@ + (nullable instancetype)imageClassifierWithOptions:(TFLImageClassifierOptions *
   [options.classificationOptions
       deleteCStringArraysOfClassificationOptions:&(cOptions.classification_options)];
 
-  if (!imageClassifier || ![TFLCommonUtils checkCError:createClassifierError toError:error]) {
```

Comment:
@khanhlvg  I guess the implementation of TFLCommonUtils was modified slightly from my version after the error refactor PR was merged. When I checked with my latest tests, this condition check was causing the error to not be populated as LHS evaluates to true and hence or condition's RHS was not being evaluated. I have modified the condition checks to cover all bases for Image Classifier and Object Detector in this PR. Image segmentation has been modified in its respective code refactor PR.

Reply:
Yes it was pointed out in the internal review so I changed it. Thanks for catching the issue and fixed it.